centreon/webpack.config.dev.js

```diff
@@ -77,6 +77,9 @@ module.exports = merge(
     plugins: devServerPlugins,
     resolve: {
       alias: {
+        '@centreon/ui/fonts': path.resolve(
+          './node_modules/@centreon/ui/public/fonts'
+        ),
```

Comment:
```suggestion
```

Reply:
does it work without? afaik. webpack needs to be configured to handle the alias